bin/BuildPackages.sh

```diff
@@ -60,36 +30,52 @@ then
         "Type: cd ../pkg; ../bin/BuildPackages.sh"
 fi
 
-# CURDIR
 CURDIR="$(pwd)"
+GAPDIR="$(cd .. && pwd)"
+COLORS=yes
+PACKAGES=
+
+# If output does not go into a terminal (but rather into a log file),
+# turn of colors.
```

Comment:
In my eyes, comments need to be useful, that's the primary measure. I don't believe in unifying for the sake of unifying. If there is a concrete comment you think that is not helpful (because its too short, too long, misses the point, etc.), please point it out.